sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/networking.py

```diff
@@ -186,6 +191,9 @@ class ServiceTagDestination(OutboundRule):
     :param port_ranges: A comma-separated list of single ports and/or range of ports, such as "80,1024-65535".
         Traffics should be allowed to these port ranges.
     :type port_ranges: str
+    :ivar address_prefixes: Optional list of CIDR prefixes or IP ranges, when provided, service_tag argument will
```

Comment:
what about the service_tag? any docstring around that?

Reply:
yes, service_tag is existing parameter and the docstring is here on line 187 

<img width="668" alt="image" src="https://github.com/user-attachments/assets/7f316ff5-003b-41c7-ad44-6b7ef57d2200">